notebook/static/notebook/js/cell.js

```diff
@@ -280,6 +286,45 @@ define([
         }
         return was_selected_cell;
     };
+    
+    /**
+     * Marks the cell
+     * @return {Cell} this
+     */
+    Cell.prototype.mark = function() {
+        if (!this.marked) {
+            this.element.addClass('marked');
+        }
+        return this;
+    };
+    
+    /**
+     * Unmarks the cell
+     * @return {Cell} this
+     */
+    Cell.prototype.unmark = function() {
+        if (this.marked) {
+            this.element.removeClass('marked');
+        }
+        return this;
+    };
+    
```

Comment:
I think the `.marked` property is enough- `cell.marked = !cell.marked;`.  I actually hesitated exposing `mark` and `unmark` as public functions because the `marked` property handles both.  What do you think, should I remove `mark` and `unmark`?

PS: If you really want to be tricky I think you can do `cell.marked ^= true;` to toggle the marked value.